### ISSUE TITLE
Misleading words about should.js in README

### ISSUE THREAD

**Issue by btd** [moll/js-must#60]
Hello.

I am a should.js maintainer and everything you described in README was covered in should.js even before must.js was created. Could you please remove should.js parts to do not mislead users?

**Comment by moll** [comment]
So everything is a function now in Should.js? I'm quite certain back in the day the `foo.should.be.false` assertion-style existed.

**Comment by btd** [comment]
Yes, i know about such problem, i was thinking about it almost from begining of maintainance. 
I tried a lot of things first to avoid breaking change (like adding plugins to jshtin/eslint, proxies to assert on non existing propety access), but eventually decide to make this breaking change - nobody complained about it.

**Comment by moll** [comment]
That's good. Since what version did you fix this? I'll make a note of that in the README that what I said applies only to previous versions. ;)

**Comment by btd** [comment]
It is safe to assume since 7.0.0. Thanks.
